runtime/src/main/java/org/corfudb/util/NodeLocator.java

```diff
@@ -50,7 +54,7 @@ public enum Protocol {
      * The host the node is located on.
      */
     @Getter
-    @NonNull
+    // @NonNull
```

Comment:
Why is this commented out?

Reply:
CorfuServer starts to listen on all interfaces when an address or network interface is not mentioned in the arguments. In such cases, the host can be null. 

CorfuServer needs a port as a mandatory argument, whereas a host is optional.